ArchiSteamFarm/Bot.cs

```diff
@@ -1173,8 +1173,21 @@ public sealed class Bot : IDisposable {
 			// We must escape our message prior to sending it
 			message = Escape(message);
 
-			for (int i = 0; i < message.Length; i += maxMessageLength) {
-				string messagePart = message.Substring(i, Math.Min(maxMessageLength, message.Length - i));
+			int i = 0;
+			int partLength;
```

Comment:
`int partLength` should be declared inside `while`, as you're not accessing it from outside the loop.